src/flow/src/lib.rs

```diff
@@ -12,8 +12,14 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//! This crate manage dataflow in Greptime, including adapter, expr, plan, repr and utils.
+//! It can transform substrait plan into it's own plan and execute it.
+//! It also contains definition of expression, adapter and plan, and internal state management.
+
 #![allow(dead_code)]
 #![allow(unused_imports)]
+#![warn(missing_docs)]
```

Comment:
Does this take the same effect like `clippy.toml`?

Reply:
> Does this take the same effect like `clippy.toml`?

`missing-docs-in-crate-items` is for config `clippy::missing_docs_in_private_items`  so to only check for missing documentation in items visible within the current crate. So it should not be a overlapping with `missing_docs`
see: https://rust-lang.github.io/rust-clippy/master/index.html#/clippy::missing_docs_in_private_items